server/worker/jira.go

```diff
@@ -1,31 +1,152 @@
 package worker
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
+	"html/template"
 
+	jira "github.com/andygrunwald/go-jira"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 )
 
-type Jira struct {
-	ds  fleet.Datastore
-	log kitlog.Logger
-	// TODO: add jira client
+const (
+	// JiraName is the name of the job as registered in the worker.
+	JiraName = "jira"
+
+	nvdCVEURL = "https://nvd.nist.gov/vuln/detail/"
+)
+
+var jiraSummaryTmpl = template.Must(template.New("").Parse(
+	`Vulnerability {{ .CVE }} detected on {{ len .Hosts }} host(s)`,
+))
+
+// jira uses wiki markup in the v2 api?
```

Comment:
Leftover comment?

Reply:
Michal added it, I think it's good to know and be aware of that, I'll just remove the interrogation point.